docs/en/stack/ml/df-analytics/dfa-classification.asciidoc

```diff
@@ -17,57 +17,72 @@ other genres available in the set of categories we have. Therefore,
 
 
 [discrete]
 [[dfa-classification-supervised]]
 ==== Training the {classification} model
 
 {classification-cap} – just like {regression} – is a supervised {ml} process. It 
-means that you need to supply a labeled training dataset that has some 
-{feature-vars} and a {depvar}. The {classification} algorithm learns the 
-relationships between the features and the {depvar}. Once you’ve trained the 
-model on your training dataset, you can reuse the knowledge that the model has 
-learned about the relationships between the data points to classify new data. 
-Your training dataset should be approximately balanced which means the number of 
-data points belonging to the various classes should not be widely different, 
-otherwise the {classanalysis} may not provide the best predictions. Read 
-<<dfa-classification-imbalanced-classes>> to learn more.
+means that you need to supply a labeled training data set that has a {depvar} 
+and some fields that are related to it. The {classification} algorithm learns 
+the relationships between these fields and the {depvar}. Once you’ve trained the 
+model on your training data set, you can reuse the knowledge that the model has 
+learned about the relationships between the data points to classify new data.
+
+The effects of imbalanced data are automatically mitigated before the 
+training. Nonetheless, it is a good idea to train your model with a data set 
+that is approximately balanced. That is to say, ideally your data set should 
+have a similar number of data points for each class.
 
 
 [discrete]
 [[dfa-classification-algorithm]]
 ===== {classification-cap} algorithms
 
 //tag::classification-algorithms[]
-The ensemble algorithm that we use in the {stack} for {classification} is a type 
-of boosting called boosted tree regression model which combines multiple weak 
-models into a composite one. We use decision trees to learn to predict the 
-probability that a data point belongs to a certain class.
+{classanalysis-cap} uses an ensemble algorithm that is a type of boosting called 
+boosted tree regression model which combines multiple weak models into a 
+composite one. It uses decision trees to learn to predict the probability that a 
+data point belongs to a certain class. A sequence of decision trees are trained 
+and every decision tree learns from the mistakes of the previous one. Every tree 
+is an iteration of the last one, hence it improves the decision made by the 
+previous tree.
 //end::classification-algorithms[]
 
 
+[discrete]
+[[dfa-classification-performance]]
+==== {classification-cap} performance
+
+As a rule of thumb, a {classanalysis} with many classes takes more time to run 
+than a binary {classification} process when there are only two classes. The 
+relationship between the number of classes and the runtime is linear.
+
+The runtime also scales approximately linearly with the number of involved 
+documents below 200.000 data points. Therefore, if you double the number of 
```

Comment:
Our style guide recommends using a comma to separate out numbers larger than 999.
```suggestion
documents below 200,000 data points. Therefore, if you double the number of 
```